docroot/profiles/custom/cgov_site/modules/custom/cgov_home_landing/config/install/field.storage.node.field_mlp_page_style.yml

```diff
@@ -0,0 +1,26 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+    - options
+id: node.field_mlp_page_style
+field_name: field_mlp_page_style
+entity_type: node
+type: list_string
+settings:
+  allowed_values:
+    -
+      value: standard
+      label: Standard
+    -
+      value: ncids
```

Comment:
can you make this value `ncids_without_title`? Keep the label, but if we ever need to add others we won't be able to until the values in prod are changed. E.g. you get stuck with this value for a long time.

Reply:
After a discussion with Bryan - we are changing the value to `ncids_default` and are keeping the label as is